### DISCUSSION TITLE
feature request

### DISCUSSION THREAD

**Discussion by DeviateBacon3** [ThioJoe/YT-Spammer-Purge#876] [General]
mayby add it so if you have the id you can put it were the spammer purge does not remove it

**Comment by Firecul** [comment]
Sorry what do you mean?

**Reply by Kurdakov** [reply]
there is whitelist feature in settings file, if it is what you need

**Reply by Firecul** [reply]
> there is whitelist feature in settings file, if it is what you need

That could be it yeah

**Comment by ThioJoe** [comment]
If you exclude a user when the results are presented, you should also be given the option to whitelist the user. I forget if the whitelist file is created before you do this, but you can go into the `SpamPurge_Resources` folder and look for `whitelist.txt`. You can just add any channel ID by itself on a line in that file, and that ID will not be included in any scan results.
